bridgetown-core/lib/bridgetown-core/concerns/site/renderable.rb

```diff
@@ -18,6 +17,13 @@ def render
       Bridgetown::Hooks.trigger :site, :post_render, self, payload
     end
 
+    # Executes inline ruby frontmatter if
```

Comment:
Thanks for adding this extra detail in here! Just a note that `ruby` should be capitalized

Reply:
You're right! I'll fix this!